### PR TITLE
Updated python3 template

### PR DESCRIPTION
**Pull request by fike**
# Pull Request

## What I did

Updated Python3 template to use 4 inputs based Golang template, issue #193 

## How I did it

Updated to 4 input variables in main.py, formula.py and config.json.

You see more than files mentioned before because I ran pre-commit.sh as Readme.md describe.

## How to verify it

Run `rit create formula`, pick python3 template and run formula based this template. You'll interact with 4 inputs.

## What kind of change does this PR make

- [ ] Major
- [X] Minor
- [ ] Patch

### CODE REVIEW

**Comment by fabianofernandeszup** [comment]
First thanks for the contribution.
We saw that your PR came with other files changed besides the Python template.
Could you adjust it so we can validate it?

**Review by fabianofernandeszup — CHANGES_REQUESTED** [review]
First thanks for the contribution.
We saw that your PR came with other files changed besides the Python template.
Could you adjust it so we can validate it?

**Comment by fike** [comment]
Certainly, @fabianofernandeszup.

**Comment by fabianofernandeszup** [comment]
By the pattern of changes in the files, I create that were made by pre-commit.
In this case, it is not necessary to run locally before the PR. We made this exception during Hacktoberfest.

**Comment by fike** [comment]
> By the pattern of changes in the files, I create that were made by pre-commit.
> In this case, it is not necessary to run locally before the PR. We made this exception during Hacktoberfest.

I cleaned files outside pythont template. Can you check again, @fabianofernandeszup?

File: templates/create_formula/languages/python3/src/formula/formula.py [hunk 1]
```diff
@@ -2,8 +2,14 @@
...
 
 
-def Run(input1, input2, input3):
+def Run(input1, input2, input3, input4):
     print("Hello World!")
-    print("%sYou receive {} in text.%s".format(input1) % (fg(2), attr(0)))
-    print("%sYou receive {} in list.%s".format(input2) % (fg(1), attr(0)))
-    print("%sYou receive {} in boolean.%s".format(input3) % (fg(3), attr(0)))
+    print("%sMy name is {}.%s".format(input1) % (fg(2), attr(0)))
+    if input2:
```
Comment (fabianofernandeszup) [inline templates/create_formula/languages/python3/src/formula/formula.py:8]: As the input2 parameter comes in a string, it is necessary to convert to boolean.
I did a test below with the conversion and it worked, what do you think of this implementation?

```
#!/usr/bin/python3
from colored import fg, attr
from distutils.util import strtobool

def Run(input1, input2, input3, input4):
    print("Hello World!")
    print("%sMy name is {}.%s".format(input1) % (fg(2), attr(0)))
    if (bool(strtobool(input2))):
        print("%sI've already created formulas using Ritchie.%s" % (fg(4), attr(0)))
    else:
        print("%sI'm excited in creating new formulas using Ritchie.%s" % (fg(1), attr(0)))
    print("%sToday, I want to automate {}.%s".format(input3) % (fg(3), attr(0)))
    print("%sMy secret is {}.%s".format(input4) % (fg(6), attr(0)))
```

Comment (fike) [reply templates/create_formula/languages/python3/src/formula/formula.py]: Thanks, @fabianofernandeszup! 

I did that!

**Review by GuillaumeFalourd — APPROVED** [review]
LGTM!

**Review by fabianofernandeszup — APPROVED** [review]

**Pull request merged**